datadog_sync/utils/configuration.py

```diff
@@ -61,14 +63,36 @@ async def init_async(self, cmd: Command):
                 try:
                     await _validate_client(self.destination_client)
                 except Exception:
-                    exit(1)
+                    sys.exit(1)
             if cmd in [Command.IMPORT, Command.MIGRATE]:
                 try:
                     await _validate_client(self.source_client)
                 except Exception:
-                    exit(1)
+                    sys.exit(1)
             self.logger.info("clients validated successfully")
 
+        # Don't sync if DDR is active
+        if self.verify_ddr_status:
+            try:
+                await _verify_ddr_status(self.destination_client)
+            except Exception as err:
+                self.logger.error(
+                    "The destination DDR verification failed. Use the --verify-ddr-status "
```

Comment:
Maybe something a bit more descriptive? Like, "The destination DDR verification failed indicating a disaster recovery is in progress"?

Reply:
Previous error messages:
```
The destination DDR verification failed. Use the --verify-ddr-status flag to override, exiting: Could not get DDR status
The destination DDR verification failed. Use the --verify-ddr-status flag to override, exiting: DDR status: ACTIVE
The destination DDR verification failed. Use the --verify-ddr-status flag to override, exiting: DDR status: FAILOVER
```

New error messages:
```
The destination DDR verification failed. This indicates that no DDR status could be retrieved. Use the --verify-ddr-status flag to override.
The destination DDR verification failed. This indicates disaster recovery is in progress. DDR status retrieved: ACTIVE  
The destination DDR verification failed. This indicates disaster recovery is in progress. DDR status retrieved: FAILOVER
```